### ISSUE TITLE
the Great Britain party set is broken (e.g. missing the Conservative Party)

### ISSUE THREAD

**Issue by mhl** [mysociety/yournextrepresentative#823]
```
In [10]: party_set = PartySet.objects.get(name='Great Britain')

In [11]: post.extra.party_set.parties.filter(name__icontains='Conservative Party')
Out[11]: [<Organization: Brentwood and Ongar Independent Conservative Party>, <Organization: The Pro Euro Conservative Party>]
```

@symroe Any idea how this could have happened?

**Comment by mhl** [comment]
Oh, no - it's been renamed. Previously:

```
In [7]: Organization.objects.get(extra__slug='party:52')
Out[7]: <Organization: Conservative Party>
```

... but on the DC site:

```
In [25]: Organization.objects.get(extra__slug='party:52')
Out[25]: <Organization: Conservative and Unionist Party>
```

**Comment by jf1** [comment]
This highlights the need to support party descriptions, ideally with the ability to select a default party description that isn't the party's primary name. Even better if country specific defaults were supported, so the appropriate description is automatically offered/pre-selected for elections in Scotland and Wales. 

According to EC the primary name was changed to "Conservative and Unionist Party" on 15/03/2016 with "Conservative Party" introduced as a description on the same day. Existing descriptions include:
Conservative Party Candidate
Scottish Conservative and Unionist
The Conservative Party Candidate
Welsh Conservative Party Candidate

**Comment by jf1** [comment]
I've just had to add two candidates whose SoPN party descriptions weren't available in the DC party list.

We mustn't underestimate the number of users who'd be confused by this, or the potential for them to abandon the immediate process and potentially abandon the site forever, so +1 to supporting party descriptions (for "Labour and Co-operative Party" etc) and a default one to override the party's primary name (especially to deal with the "Conservative and Unionist" situation)

**Comment by jf1** [comment]
I've since seen "Labour and Co-operative Party" listed as a party in DC's data - this isn't technically correct but I'm not sure how best to address this.

There is no party with a primary name of "Labour and Co-operative Party" but the two separate parties  each have this listed as a description, qualified by "(Joint Description with [other] Party)" - thereby making it impossible to for these candidates to be assigned to **a** party.
http://search.electoralcommission.org.uk/English/Registrations/PP53
http://search.electoralcommission.org.uk/English/Registrations/PP119

DC could list "Description" (rather than "Party") for all candidates, with a grouping system to link a party's candidates together, but that still wouldn't deal with candidates who are standing on behalf of more than one party - suggestions appreciated! Arguably such candidates should be grouped into the statistics for **both** the Labour **and** Co-operative part**ies** but that'd mean any page listing all party totals would need to highlight that X candidates are standing on behalf of more than one party, hence the sum of party totals being X higher than the number of candidates.

**Comment by andylolz** [comment]
@jf1: yeah – “joint descriptions pseudoparties” have been manually added to YNR (everything else is scraped from the Electoral Commission). There aren’t many (as I recall, TUSC/Left Unity and Labour/Co-op were the main ones in 2015) so manually adding them as pseudoparties is a quick and hacky solution to capture the fact that they’re standing for both parties.

**Comment by mhl** [comment]
I think this has been partly fixed now by @symroe's addition of the EC registered descriptions to the party drop-downs on the bulk adding page. It's an open question whether we should also display these in the drop-down on the normal candidate create / edit pages. (I think we probably should, even at the risk that it might confuse people that that descriptions aren't actually stored (because someone creating a party and picking the description "Conservative Party" will then see them as "Conservative and Unionist" once the person is created).)
